src/initial_data.json

```diff
@@ -3,8 +3,8 @@
     "model": "sites.site",
     "pk": 1,
     "fields": {
-      "domain": "localhost",
-      "name": "localhost"
+      "domain": "bytedeck.com",
+      "name": "bytedeck.com"
```

Comment:
This should stay as localhost?  I'm not worried about setting up other production environments, the app should work as close as possible "out of the box" to set up a development environment.  The production environment is the special case where we can manually change values, the defaults should be for development environment and then overridden in prorduction (via the production settings file, .env, or otherwise)

Reply:
This will never get used again on this production environment, once it is run the first time, right?